validator/src/main/java/com/google/daq/mqtt/util/ConfigManager.java

```diff
@@ -275,4 +279,26 @@ public String getUpdatedTimestamp() {
     return isoConvert(metadata.timestamp);
   }
 
+  private DiscoveryConfig getDiscoveryConfig() {
+    DiscoveryConfig discoveryConfig = new DiscoveryConfig();
+    discoveryConfig.families = new HashMap<>();
```

Comment:
I try to be pedantic with some of this and only set values that are defined in the source.  Theoretically (future proofing) there might be a DiscoveryModel that does NOT have a families block, and in that case this should be null as well. There's an easy construct we have to do that, something like:

discoveryConfig.families = ifNotNullGet(metadata.discovery.families, families --> new HashMap());

basically, at each level of the hierarchy, there should be a config entry If-and-only-if there's a corresponding metadata entry!